requirements.txt

```diff
@@ -0,0 +1,120 @@
+appdirs==1.4.3
+apturl==0.5.2
+autopep8==1.5.7
+backports.entry-points-selectable==1.1.0
+bcrypt==3.1.7
+binwalk==2.2.0
+blinker==1.4
+Brlapi==0.7.0
+certifi==2019.3.9
+chardet==3.0.4
+chrome-gnome-shell==0.0.0
+Click==7.0
+colorama==0.4.3
+command-not-found==0.3
+cryptography==2.8
+cupshelpers==1.0
+cycler==0.10.0
+dbus-python==1.2.16
+decorator==4.4.2
+defer==1.0.6
+distlib==0.3.2
+distro==1.4.0
+distro-info===0.23ubuntu1
```

Comment:
You can use or virtual environments to keep only requirements specific to your project otherwise you deployment will be heavy in size and cost.